packages/launchpad/cypress/e2e/migration.cy.ts

```diff
@@ -206,6 +214,94 @@ describe('Full migration flow for each project', { retries: { openMode: 2, runMo
     finishMigrationAndContinue()
   })
 
+  it('skips the file renaming migration-e2e-component-default-test-files', () => {
```

Comment:
See https://github.com/cypress-io/cypress/pull/20145/files#r804342392 - can we group tests for the same projects with `context`? We have a lot of tests (good) but we need to organize them.